tap/api/api.go

```diff
@@ -117,6 +122,7 @@ type MizuEntry struct {
 type MizuEntryWrapper struct {
 	Protocol       Protocol  `json:"protocol"`
 	Representation string    `json:"representation"`
+	BodySize       int64     `json:"body_size"`
```

Comment:
Same camelCase as elapsed_time comment

Reply:
@gadotroee I'm sorry, I'm used to snake_case when I see a JSON field. Fixed it with https://github.com/up9inc/mizu/commit/516a3e59b062f864ad54f5d7ee3a4f8e4a8a2259